src/main.py

```diff
@@ -72,3 +77,10 @@
     # get top k terms with the highest idf score (this is atm not very useful
     # but it will be used for the tf.idf similarity stuff
     top_keywords = get_top_idf_features(train_description_feature, 5)
+
+    # calculate similarity between train an testset job descriptions
+    # this is of high order complexity - test it on a subset of the data
+    corpus_list = pandas_vector_to_list(train_description_feature)
+    queries_list = pandas_vector_to_list(test_description_feature)
+    similarity = get_tfidf_similarity(corpus_list, queries_list)
+    relevant_documents = extract_relevant_documents(similarity)
```

Comment:
@utku yea i agree
as mentioned, clustering is not what we want here i thing.. i.e. we don't just want to throw all software engineers in one pot and as soon as there are in london and are software engineers they get X salary.

using the similarity thing here, you want account for fine nuances and differentiate on a very low level e.g. between a javascript and a php web developer